sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -424,7 +424,7 @@ object ScalaNativePluginInternal {
         if (exitCode == 0) None
         else Some("Nonzero exit code: " + exitCode)
 
-      Defaults.toError(message)
```

Comment:
Why this change?

Reply:
That's to fix the deprecation warning:
https://github.com/sbt/sbt-zero-thirteen/blob/3d50ca2f6ab531ac0c5ca23b1ff2b40cda84300a/main/src/main/scala/sbt/Defaults.scala#L2200-L2201

Reply:
Sounds good then!